src/components/Toast/Toast.test.tsx

```diff
@@ -259,6 +253,7 @@ describe('Toast', () => {
 
       act(() => {
         jest.advanceTimersByTime(1000);
+        jest.useRealTimers();
```

Comment:
@RyanMacBern curious why we need to manually set jest to useRealTimers if this runs after every test.

Reply:
This test started failing after the upgrade. I spent quite a bit of time messing with it, and this is the only thing that worked. For whatever reason, it needs to switch to real timers **before** the last expect or the text will not be in the document.